### DISCUSSION TITLE
How to perform a clean install?

### DISCUSSION THREAD

**Discussion by roshin8** [outdoorbits/little-backup-box#237] [Q&A] [answered]
I installed it once but installation of  comitup got messed up
So now i want to install everything all over again but everytime I run the installer script, it doesn't ask me where i want the default backup to be. How to clear configs that are already saved so it doesn't use old settings

**Comment by roshin8** [comment, marked as answer]
I figured out by looking at the install script. Remove the folder `/var/www/little-backup-box` and run the installer again

**Comment by outdoorbits** [comment]
I did a new installation from scratch two days ago, I didn't have any problems, even not about Comitup. If you like, you can try to install Comitup by the installer script in /home/pi/little-backup-box/
